### ISSUE TITLE
Creating a debian 8 profile 

### ISSUE THREAD

**Issue by Robertgreen78** [volatilityfoundation/volatility#430]
Im having a problem making a debian 8 profile i would be realy glad if someone can help me
or make one for me
thanks,
robert

**Comment by pagabuc** [comment]
I think before being able to create a debian 8 profile, you should really read something like [this](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html). What does not work? What are the steps to reproduce? What is the output of the program? How did you acquire the memory dump?

**Comment by openfoamtutorials** [comment]
I made a custom profile for Ubuntu 16.04.2 (only up to 16.04.1 was available). I first had to install a dependency or two (dwarfdump and maybe another, but it will be apparent), then run "make" inside "volatility/tools/linux". Then, just zip these 2 files together:
1. module.dwarf (result of the "make" build)
2. /boot/System.map...
And put the zip file into "volatility/plugins/overlays/linux/". The profile is now added.
Now "python vol.py --info" should show the "--profile" value associated with the profile you just added.
